src/components/Sidebar.vue

```diff
@@ -1,56 +1,59 @@
 <template>
   <div
-    @mouseover="hover = true"
-    @mouseleave="hover = false"
-    :class="[hover ? 'sidebar' : 'sidebar collapsed-sidebar']"
+    @mouseover="expanded = true"
+    @mouseleave="expanded = false"
+    :class="[expanded ? 'sidebar expanded' : 'sidebar collapsed-sidebar']"
```

Comment:
I think this is possible:
```suggestion
    :class="['sidebar', expanded ? 'expanded' : 'collapsed-sidebar']"
```